src/modules/discovery/actions.js

```diff
@@ -18,3 +17,15 @@ const performGohanGeneSearch = networkAction((searchUrl) => () => ({
     url: searchUrl,
     err: "error performing Gohan gene search",
 }));
+
+const _fetchDiscoverySchema = networkAction(() => (dispatch, getState) => ({
+    types: FETCH_DISCOVERY_SCHEMA,
+    url: `${getState().bentoServices.itemsByKind.metadata.url}/api/discovery_schema`,
+    err: "Error fetching discovery JSON schema",
+}));
+
+export const fetchDiscoverySchema = (dispatch, getState) => {
```

Comment:
this module was eliminated in the perms pt. 2 PR, so it'll lead to conflicts - we can reintroduce it, or move this to the metadata module.

Reply:
moved into metadata